From 100fd82daf917a705d2098d339797c8c5ab732ef Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Sun, 1 Feb 2015 18:11:16 +0800
Subject: [PATCH 0946/5242] MLK-10190 ARM: dts: disable disp mix on imx6sl by
 default

commit  6e59e098dc4fa5ebfc5c5d33084da8b51af652bb from
https://source.codeaurora.org/external/imx/linux-imx.git

The cause is EPDC works not stable if DISP mix is enabled.

Signed-off-by: Robby Cai <r63905@freescale.com>
(cherry picked from commit db5b89bd35d259504da1c15d62f898f1291541e2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl.dtsi |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sl.dtsi b/arch/arm/boot/dts/imx6sl.dtsi
index 94331c5..3719861 100644
--- a/arch/arm/boot/dts/imx6sl.dtsi
+++ b/arch/arm/boot/dts/imx6sl.dtsi
@@ -768,7 +768,6 @@
 					<&clks IMX6SL_CLK_DUMMY>,
 					<&clks IMX6SL_CLK_DUMMY>;
 				clock-names = "disp-axi", "csi_mclk", "disp_dcic";
-				power-domains = <&gpc 2>;
 				status = "disabled";
 			};
 
@@ -796,7 +795,6 @@
 				interrupts = <0 98 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SL_CLK_PXP_AXI>, <&clks IMX6SL_CLK_DUMMY>;
 				clock-names = "pxp-axi", "disp-axi";
-				power-domains = <&gpc 2>;
 				status = "disabled";
 			};
 
@@ -806,7 +804,6 @@
 				interrupts = <0 97 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SL_CLK_EPDC_AXI>, <&clks IMX6SL_CLK_EPDC_PIX>;
 				clock-names = "epdc_axi", "epdc_pix";
-				power-domains = <&gpc 2>;
 			};
 
 			lcdif: lcdif@20f8000 {
@@ -817,7 +814,6 @@
 					 <&clks IMX6SL_CLK_LCDIF_AXI>,
 					 <&clks IMX6SL_CLK_DUMMY>;
 				clock-names = "pix", "axi", "disp_axi";
-				power-domains = <&gpc 2>;
 				status = "disabled";
 			};
 
-- 
1.7.9.5

